src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -213,10 +213,22 @@ def list_releases(target_name, target_resource_group):
     json_request = req.json()
     return json_request
 
+def _is_inside_git_directory():
+    """
+    Determines if the user is inside the .git folder of a git repo
+    """
+    try:
+        is_inside_git_dir = check_output(['git', 'rev-parse', '--is-inside-git-dir'])
+    except OSError:
+        raise CLIError('Git is not currently installed.')
+    return is_inside_git_dir.decode('utf-8').strip()
```

Comment:
I just tried the command you run and got the following:
```
$ git rev-parse --is-inside-git-dir
false
$ cd .git
$ git rev-parse --is-inside-git-dir
true
$ 
```
So the git command returns a string.

This method should return a boolean rather than a string.
i.e. you could have something like:
```
git_result = is_inside_git_dir.decode('utf-8').strip()
if git_result == 'false':
    return False
elif git_result == 'true':
    return True
else:
    raise CLIError(...)
```